### ISSUE TITLE
ISE script execution fails for user in developer role

### ISSUE THREAD

**Issue by michaellwest** [SitecorePowerShell/Console#342]
Created a new user and added to the developer role using this code as an admin:

```powershell
New-User -Identity michael -Password b -Enabled
Add-RoleMember -Identity developer -Members michael
```

After logging as the new user, I launched the ISE and executed the default script. I received the following error:

```
The current user does not have the required rights to perform the operation: 
'Copy item ({DB19F00D-05F0-4589-8807-189CE2807224} -> {1FBA8C81-EDD3-4133-B50B-13C873F79827})'. 
User: sitecore\michael (method: Sitecore.Data.Items.Item.CopyTo(Item destination, String copyName, ID copyID, Boolean deep)).
```

The error occurs at line 201 of the file `c:\inetpub\wwwroot\Console\Core\Settings\ApplicationSettings.cs`

**Comment by AdamNaj** [comment]
Fixed :)

**Comment by AdamNaj** [comment]
Merged by implementing Issue #350
